roles/metrics-export.lua

```diff
@@ -170,14 +168,48 @@ local function validate_http_endpoint(endpoint)
     end
 end
 
+-- check_server_httpd_role validates that httpd configuration and provided name exists.
+local function check_server_httpd_role(server)
+    local httpd_roles_cfg = require("config"):get("roles_cfg")['roles.httpd']
+    if httpd_roles_cfg == nil then
+        error("there is no configuration for httpd role")
+    end
+    if httpd_roles_cfg[server] == nil then
+        error(('server with name %s not found in httpd role config'):format(server))
+    end
+end
+
+-- get_server_from_httpd_role_cfg returns http server by provided server_name from httpd role.
+-- If server_name not found returns server by DEFAULT_SERVER_NAME.
+local function get_server_from_httpd_role_cfg(server_name)
+    local httpd_roles_cfg = require("config"):get("roles_cfg")['roles.httpd']
+    return httpd_roles_cfg[server_name] or
+           httpd_roles_cfg[httpd_role.DEFAULT_SERVER_NAME]
```

Comment:
httpd_role.DEFAULT_SERVER_NAME should be used only if server_name == nil.

The code is confusing at now.
```suggestion
local function get_server_from_httpd_role_cfg(server_name)
    server_name = server_name or httpd_role.DEFAULT_SERVER_NAME
    local httpd_roles_cfg = require("config"):get("roles_cfg")['roles.httpd']
    return httpd_roles_cfg[server_name]
```

Reply:
Corrected, thank you.